docs/clients/cognito-idp.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: client
+category: clients
+name: CognitoIdentityProvider
+package: async-aws/cognito-identity-provider
+---
+
+## Usage
+
+
+### Create a user as an admin
+
+```php
+use AsyncAws\CognitoIdentityProvider\CognitoIdentityProviderClient;
+use AsyncAws\CognitoIdentityProvider\ValueObject\AttributeType;
+
+$cognitoIdp = new CognitoIdentityProviderClient();
+
+$cognitoIdp->adminCreateUser([
+    'UserPoolId' => 'us-east-1_1337oL33t',
```

Comment:
Is this a real pool name?

Reply:
no

Reply:
We should use pool name that look real. 

Like "website"?

Reply:
This one looks real in fact, they're always composed of the region and some identifier string.

Reply:
Okey. Good.